ipn/ipnlocal/local.go

```diff
@@ -27,9 +27,10 @@ import (
 	"sync/atomic"
 	"time"
 
+	"slices"
```

Comment:
Group with the rest of the stdlib? (Usually fmt is better about this)

Reply:
Whoops. I thought I'd gotten them all already.